pkgs/development/compilers/llvm/5/clang/default.nix

```diff
@@ -0,0 +1,97 @@
+{ stdenv, fetch, cmake, libxml2, libedit, llvm, version, release_version, clang-tools-extra_src, python
+, fixDarwinDylibNames
+, enableManpages ? false
+}:
+
+let
+  gcc = if stdenv.cc.isGNU then stdenv.cc.cc else stdenv.cc.cc.gcc;
```

Comment:
Just noticed this, `cc.cc.gcc` doesn't exist on other cc wrappers. Since it's only referenced on linux I think it's better to just inline it.